### PR TITLE
Directory dependence fix

### PR DESCRIPTION
**Pull request by andrei4ka**
Directory dependence fix

### CODE REVIEW

**Comment by andrei4ka** [comment]
Please accept my changes.

File: deployment/puppet/osnailyfacter/manifests/tinyproxy.pp [hunk 1]
```diff
@@ -7,11 +7,23 @@
...
-      /etc/init.d/tinyproxy restart; ",
+    /sbin/chkconfig tinyproxy on; 
+    /etc/init.d/tinyproxy restart; ",
+    }
   }
+  'Debian': {
+    exec{'tinyproxy-init':
+    command => "/bin/echo Allow $master_ip >> /etc/tinyproxy.conf; 
+    /usr/sbin/update-rc.d tinyproxy defaults; 
+    /etc/init.d/tinyproxy restart; ",
```
Comment (xdel) [inline deployment/puppet/osnailyfacter/manifests/tinyproxy.pp]: It would be awesome to move to Exec ~> Service instead of one large exec.

Comment (andrei4ka) [inline deployment/puppet/osnailyfacter/manifests/tinyproxy.pp]: In the next commit =) And now accept the changes

Comment (andrei4ka) [inline deployment/puppet/osnailyfacter/manifests/tinyproxy.pp]: ok i`ll do it

Comment (xdel) [inline deployment/puppet/osnailyfacter/manifests/tinyproxy.pp]: Nevermind please, tinyproxy scheduled for removal. Check the rest of services and packages for proper work instead.

**Comment by plechenko** [comment]
add to whitelist

**Comment by xdel** [comment]
Please clean it up and make mergeable.

**Comment by xdel** [comment]
retest this please

**Pull request merged**